dockerfiles/frameworks/contrib/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN set -xe; \
         php${V}-dom \
         php${V}-gd \
         php${V}-iconv \
-        php${V}-imagick \
+        # php${V}-imagick \
```

Comment:
Should we remove this one?

Reply:
Don't know - the reason is that recent versions of imagick do understand pdf, which makes the older wordpress testsuites fail.

Reply:
```suggestion
```
Let's remove it then! If it fails in the future we'll get back to it